CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 0.40.0
 
+* Migrate from OpenTelemetry tracing from using OkHttp sender to the JDK sender or GRPC sender thus remove the kotlin dependency. 
```

Comment:
This has no meaning for the users. If it has any user impact, please explain it. If not, please remove it.

Reply:
Nope there is no user impact, I have removed.